src/services/expensesService.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const createId = require('../utils/createId');
+
+const expenses = [];
+
+function init() {
+  expenses.length = 0;
+}
+
+const get = (params) => {
+  let filteredExpenses = expenses;
+
+  const { userId, categories, from, to } = params;
+
+  if (userId) {
+    filteredExpenses = filteredExpenses.filter(
+      (expense) => expense.userId === +userId,
+    );
+  }
+
+  if (categories) {
+    filteredExpenses = filteredExpenses.filter(
+      (expense) => expense.category === categories,
```

Comment:
The `categories` filter assumes `categories` is a single value. If `categories` can be an array, modify the filter to handle arrays correctly.